jsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+	"compilerOptions": {
+		"allowSyntheticDefaultImports": true,
+		"baseUrl": "."
```

Comment:
it depends on the scope in which you open the folder - jsconfig.json is used by vscode, its a microsoft thingy. So if you open a folder from a different scope in the workspace this is supposedly used for resolving paths and the like. Saying that, it may well be redundant.